test/jdk/java/lang/invoke/MethodHandleProxies/WithSecurityManagerTest.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright (c) 2023, Oracle and/or its affiliates. All rights reserved.
+ * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+ *
+ * This code is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 only, as
+ * published by the Free Software Foundation.
+ *
+ * This code is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+ * version 2 for more details (a copy is included in the LICENSE file that
+ * accompanied this code).
+ *
+ * You should have received a copy of the GNU General Public License version
+ * 2 along with this work; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+ * or visit www.oracle.com if you need additional information or have any
+ * questions.
+ */
+
+/* @test
+ * @bug 6983726
+ * @build Untrusted WithSecurityManagerTest
+ * @run main/othervm/policy=jtreg.security.policy WithSecurityManagerTest
+ * @summary Checks MethodHandleProxies behavior with security manager present
+ */
+
+import java.lang.invoke.MethodHandleProxies;
+import java.lang.invoke.MethodHandles;
+
+public class WithSecurityManagerTest {
+    public static void main(String... args) {
+        var originalMh = MethodHandles.zero(void.class);
+        Object o = MethodHandleProxies.asInterfaceInstance(Untrusted.class, originalMh);
```

Comment:
Is it better to define an interface in this test?   It seems that this test has nothing to do with Untrusted.  It may confuse readers.